api/envoy/config/filter/accesslog/v2/accesslog.proto

```diff
@@ -150,3 +153,9 @@ message HeaderFilter {
   // check.
   envoy.api.v2.route.HeaderMatcher header = 1 [(validate.rules).message.required = true];
 }
+
+// Filters requests that received responses with an Envoy response flag set.
+// A list of the response flags can be found
+// in the access log formatter :ref:`documentation<config_access_log_format_response_flags>`.
+message ResponseFlagFilter {
+}
```

Comment:
alternatively I thought about adding an option to specify the flags you care about. But I think that response flags are important, and infrequent enough to have the filter filter on all of them. Thoughts?

Reply:
One way to achieve both settings is have an array field that will specify which response flags to log on and if that field is empty default to all response flags? (or have an enum that says all)

Reply:
Yeah +1 to what @ccaraman said. I would do that.

Reply:
Yep, that is what I thought, will add.